vendor/priors/gwt.js

```diff
@@ -397,7 +402,9 @@ function getFileHandleCreateIfNecessary(filename, directory) {
             blockDirHandle.getFileHandle(filename, { create: true }));
 }
 let pendingWrites = new Map();
-
+let pendingReads = new Map();
```

Comment:
Can you comment with what the types are here? If the values are lists, why can't be a single future?

Reply:
If I have a pendingReads map<String, Future<byte[]>> then if n other reads come in can't I just call map.get(k).thenComplete(ourFuture) which would complete all waiting futures as soon as the first one is done?
So no need for a list? Maybe I'm being dense.